### DISCUSSION TITLE
Wizard WDAC fila hash rule grabs file's full path from user's folders

### DISCUSSION THREAD

**Discussion by SlyInH3ll** [MicrosoftDocs/WDAC-Toolkit#346] [General]
When creating a file name based rule, you get the option to edit the path to use wildcards, etc. When creating a file hash, the file's full path is grabbed without the option to edit. and when the file is from the user's profile structure, it can't be deployed like that on several computers, the XML need to be edited, to correct the FriendlyName and remove the path, keeping only the file name. This behaviour prompt unwanted manual interventions prone to introduce errors in the policy. It would be nice to have an option to edit file name/ file path before creating the rule, cleaning it from the start.

**Comment by jgeurten** [comment]
This is a good suggestion, @SlyInH3ll. Would it meet your requirements if I replaced the user's profile with %userprofile% and tied it to this setting in the Wizard? 

![image](https://github.com/MicrosoftDocs/WDAC-Toolkit/assets/23045608/0d092af0-60b4-4b8d-83ee-c10b194d9e6e)


When this setting is checked the following FriendlyName would be replaced with: 

FriendlyName="C:\Users\jordan\AppData\Local\*"

would be replaced with: 

FriendlyName="C:\%userprofile%\AppData\Local\*"
